### PR TITLE
Remove level from Script Edit GUI

### PR DESCRIPTION
**Pull request by dmcavoy**
Removes levels from the Script Edit GUI and moves the UI components for level editing into the lesson-editors folder. This prepares for levels moving to the lesson edit page. 

Screen Shot 2020-09-04 at 4 27 11 PM

## Testing story

The Script Edit GUI is currently not functional so I did not get the script edit gui to a working save point but I made sure the normal page that editors use was not impacted by this change by checking that that still saves.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/levelbuilder/script-editor/LessonGroupCard.jsx [hunk 1]
```diff
@@ -122,9 +122,6 @@ class LessonGroupCard extends Component {
                     this.props.setLessonMetrics(metrics, lesson.position);
                   }
                 }}
-                lessonMetrics={this.props.lessonMetrics}
```
Comment (dmcavoy) [inline apps/src/lib/levelbuilder/script-editor/LessonGroupCard.jsx:125]: There are more things here to pull out related to levels

File: apps/test/unit/code-studio/editorReduxTest.js [hunk 2]
```diff
@@ -70,29 +36,6 @@ describe('editorRedux reducer tests', () => {
   let initialState;
   beforeEach(() => (initialState = getInitialState()));
 
-  it('reorder levels', () => {
```
Comment (davidsbailey) [inline apps/test/unit/code-studio/editorReduxTest.js:73]: is there somewhere in the lesson editor this test case could be added?

**Review by davidsbailey — APPROVED** [review]
it seems like `levelbuilder/shapes.jsx` could also be moved to `lesson-editor/` at this time, since it only contains `levelShape`

File: apps/test/unit/code-studio/editorReduxTest.js (hunk 2 above)
Comment (dmcavoy) [reply apps/test/unit/code-studio/editorReduxTest.js:73]: See https://github.com/code-dot-org/code-dot-org/pull/36625

**Comment by uponthesun** [comment]
Just commenting to say that I don't think I'll have much input to offer on this.

**Pull request merged**